### PR TITLE
Added policy-integrity-shield.yaml

### PR DESCRIPTION
**Pull request by yuji-watanabe-jp**
@ycao56 this is the policy to enable Integrity Shield on managed clusters. We have tested on our test clusters. Please review and merge. Thanks.

### CODE REVIEW

File: community/CM-Configuration-Management/policy-integrity-shield.yaml [hunk 1]
```diff
@@ -0,0 +1,207 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-integrity-shield
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-5 Access Restrictions for Change
+spec:
+  remediationAction: enforce
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-integrity-shield.yaml]: update it to `inform` as we don't want to surprise users

File: community/CM-Configuration-Management/policy-integrity-shield.yaml [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-integrity-shield-namespace
+        spec:
+          remediationAction: enforce
+          severity: High
+          namespaceSelector:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-integrity-shield.yaml]: remove `namespaceSelector` as the embedded CR is a cluster scope resource

File: community/CM-Configuration-Management/policy-integrity-shield.yaml [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+                      interval: 45m
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-integrity-shield-sub
+        spec:
+          remediationAction: enforce
+          severity: high
+          namespaceSelector:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-integrity-shield.yaml]: remove `namespaceSelector` as namespace has been already specified in embedded object

File: community/CM-Configuration-Management/policy-integrity-shield.yaml [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-integrity-shield-cr
+        spec:
+          remediationAction: enforce
+          severity: high
+          namespaceSelector:
+            exclude: ["kube-*"]
+            include: ["integrity-shield-operator-system"]
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-integrity-shield.yaml]: can just specify the namespace in `integrity-shield-server`

**Review by ycao56 — APPROVED** [review]
LGTM. Will merge after @dockerymick reviewed the text

File: community/README.md [hunk 5]
```diff
@@ -82,7 +82,7 @@ Policy  | Description | Prerequisites
...
 [Trusted Node policy](./CM-Configuration-Management/policy-trusted-node.yaml) | Use the trusted node policy to detect if there are untrusted or unattested nodes in the cluster. | [Trusted Node Policy Controller](https://github.com/lumjjb/trusted-node-policy-controller)
 [ETCD Backup](./CM-Configuration-Management/policy-etcd-backup.yaml) | Use the ETCD Backup policy to receive the last six backup snapshots for etcd. This policy uses the etcd container image in the policy because it contains all required tools like etcdctl. | For more information, see [OpenShift 4 with default storage class](https://docs.openshift.com/container-platform/4.5/storage/dynamic-provisioning.html).
-
+[Integrity Shield](./CM-Configuration-Management/policy-integrity-shield)| Use the Integrity Shield to protect integrity of Kubenernetes resources in a cluster (e.g. OpenShift). | See the [Integrity Shield documentation.](https://github.com/open-cluster-management/integrity-shield/blob/master/docs/ACM/README_ENABLE_ISHIELD_PROTECTION_ACM_ENV.md)
```
Comment (dockerymick) [inline community/README.md]: The link for Integrity Shield is broken. i received a 404 Error when I selected it.

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: community/README.md (hunk 5 above)
Comment (gajananan) [reply community/README.md]: @dockerymick I fixed the link for Integrity Shield,  please check. Thank you.

File: community/README.md [hunk 6]
```diff
@@ -82,7 +82,7 @@ Policy  | Description | Prerequisites
...
 [Trusted Node policy](./CM-Configuration-Management/policy-trusted-node.yaml) | Use the trusted node policy to detect if there are untrusted or unattested nodes in the cluster. | [Trusted Node Policy Controller](https://github.com/lumjjb/trusted-node-policy-controller)
 [ETCD Backup](./CM-Configuration-Management/policy-etcd-backup.yaml) | Use the ETCD Backup policy to receive the last six backup snapshots for etcd. This policy uses the etcd container image in the policy because it contains all required tools like etcdctl. | For more information, see [OpenShift 4 with default storage class](https://docs.openshift.com/container-platform/4.5/storage/dynamic-provisioning.html).
-
+[Integrity Shield](./CM-Configuration-Management/policy-integrity-shield.yaml)| Use the Integrity Shield to protect integrity of Kubenernetes resources in a cluster (e.g. OpenShift). | See the [Integrity Shield documentation.](https://github.com/open-cluster-management/integrity-shield/blob/master/docs/ACM/README_ENABLE_ISHIELD_PROTECTION_ACM_ENV.md)
```
Comment (dockerymick) [inline community/README.md]: Update the first sentence to read: "Use the Integrity Shield to protect **the** integrity of Kubenernetes..."

**Review by dockerymick — APPROVED** [review]
Please address my comment before merging. Other than that, the changes look good !

File: community/README.md (hunk 6 above)
Comment (gajananan) [reply community/README.md]: Thank you, I have addressed your comments. @dockerymick

Comment (dockerymick) [reply community/README.md]: Thank you ! @gajananan

**Pull request merged**